### PR TITLE
MCLandDetector: clean up logic for ground effect

### PR DESCRIPTION
**Pull request by sfuhrer**
Before: in_ground_effect was always true with the default settings (LNDMC_ALT_GND =-1)
 and when a distance sensor was present with valid data.
Now: default of LNDMC_ALT_GND is set to 2m by default, and if set to a negative value
then in_ground_effect is never set to true

### CODE REVIEW

**Review by bresch — DISMISSED** [review]

File: src/modules/land_detector/MulticopterLandDetector.cpp [hunk 1]
```diff
@@ -188,7 +188,7 @@ bool MulticopterLandDetector::_get_ground_contact_state()
 	}
 
 	if (lpos_available && _vehicle_local_position.dist_bottom_valid) {
```
Comment (MaEtUgR) [inline src/modules/land_detector/MulticopterLandDetector.cpp:190]:
```suggestion
	if (lpos_available && _vehicle_local_position.dist_bottom_valid && _param_lndmc_alt_gnd_effect.get() > 0) {
```
Maybe just to be sure if there's a misbehaving distance sensor driver that publishes a negative value.

**Review by MaEtUgR — APPROVED** [review]
Thanks for addressing this 👍

File: src/modules/land_detector/MulticopterLandDetector.cpp (hunk 1 above)
Comment (sfuhrer) [reply src/modules/land_detector/MulticopterLandDetector.cpp]: okay makes sense, added it

**Review by MaEtUgR — APPROVED** [review]
👍 ✨

**Pull request merged**